core/test/scripts/Voting.js

```diff
@@ -22,6 +22,46 @@ class MockNotifier {
   }
 }
 
+const TEST_IDENTIFIERS = {
+  test: {
+    key: web3.utils.utf8ToHex("test"),
+    expectedPrice: web3.utils.toWei("1.5")
+  },
+  "test-bad-reveal": {
+    key: web3.utils.utf8ToHex("test-bad-reveal")
+  },
+  "test-overlapping1": {
+    key: web3.utils.utf8ToHex("test-overlapping1"),
+    expectedPrice: web3.utils.toWei("1.5")
+  },
+  "test-overlapping2": {
+    key: web3.utils.utf8ToHex("test-overlapping2"),
+    expectedPrice: web3.utils.toWei("1.5")
+  },
+  TSLA: {
+    key: web3.utils.utf8ToHex("TSLA"),
+    hardcodedTimestamp: "1573513368",
+    expectedPrice: web3.utils.toWei("345.07")
+  },
+  "Custom Index (1)": {
+    key: web3.utils.utf8ToHex("Custom Index (1)"),
+    expectedPrice: web3.utils.toWei("1")
+  },
+  "Custom Index (100)": {
+    key: web3.utils.utf8ToHex("Custom Index (100)"),
+    expectedPrice: web3.utils.toWei("100")
+  },
+  "0.5": {
+    key: web3.utils.utf8ToHex("0.5"),
+    expectedPrice: web3.utils.toWei("0.5")
+  }
+};
+
+const getRandomTimestampInPast = () => {
+  const salt = Math.floor(Math.random() * 100);
```

Comment:
Can you explain all this math (this line and below)?

Is it preferable to something like:
```
Math.floor(currentTime * Math.random())
```
?

Reply:
there really is no reason why I can't make this cleaner. I was basically taking the current timestamp, converting to seconds, subtracting a random number [0,100]